docs/sitemap-custom.xml

```diff
@@ -0,0 +1,7 @@
+<urlset>
```

Comment:
are you sure this file is served correctly, without adding it to `mkdocs.yml`?

Reply:
yes, it's available in the PR preview of the docs, should be correct this way